src/Akka.Logger.Serilog/SerilogLoggingAdapter.cs

```diff
@@ -137,18 +114,18 @@ public ILoggingAdapter SetContextProperty(string name, object value, bool destru
             var contextNode = new ContextNode
             {
                 Enricher = contextProperty,
-                Next = enricherNode
+                Next = _enricherNode
             };
 
-            return new SerilogLoggingAdapter(adapter, contextNode);
+            return new SerilogLoggingAdapter(_bus, _logSource, _logClass, contextNode);
```

Comment:
Preserved the original functionality that was added by @arturosevilla for creating semantic loggers. Didn't want to throw the baby out with the bath water.